clients/java/src/test/java/io/zeebe/client/assertions/TopologyAssert.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe.client.assertions;
+
+import io.zeebe.client.api.response.BrokerInfo;
+import io.zeebe.client.api.response.Topology;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.assertj.core.api.AbstractAssert;
+
+public class TopologyAssert extends AbstractAssert<TopologyAssert, Topology> {
```

Comment:
I think it would fit there. However, then `zeebe-test-util` would have to depend on `zeebe-client-java` which defines `Topology`. Not sure if this is intended. Please advise.

Reply:
That's a good point. I don't think the goal of this module were ever clearly defined. What would be the downsides here? The only one I can think of is that the client could not later depend on `zeebe-test-util` if we ever needed it. I guess it's a trade off - having `zeebe-test-util` vs the test JAR. I would prefer `zeebe-test-util` personally, and we can always refactor this kind of thing out into an e2e utility module if we have a need for it later on (as that's really where it should be, I guess).

What do you think?